src/app/pad.component.html

```diff
@@ -136,6 +136,17 @@
             <a class="invert" href="https://github.com/blindpad/blindpad/issues/new">feedback</a>
             <a class="invert" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=VW2QSSP76L36N">donate</a>
             <a class="invert" routerLink="/about">about</a>
+
+            <button class="keymap-button" *ngIf="!!getPadKeymap()" (click)="onKeymapButtonClick()">
+                {{ getPadKeymap() }}
+            </button>
+            <keymap-choices [@fadeInOut] *ngIf="visibleKeymapChoices != null">
+                <keymap-choice *ngFor="let keymap of visibleKeymapChoices" (click)="onKeymapChoice(keymap)">
```

Comment:
why did you ditch the `is-picked` class? it makes it kinda nice to see a checkmark or whatever next to the selected one.